README.md

```diff
@@ -111,6 +111,19 @@ It gets your certifications from the API and asks you to record which languages
 
 The `setup` command creates a configuration file which is stored in your home folder. You find it here `~/.urcli_config`.
 
+## For using this API on Android Device:
```

Comment:
Since this is a guide of sorts, I think the headline should include that. Also, it's not an API but a CLI 😉 . How about: `## How to set up and run urcli on Android`?